packages/flutter_tools/gradle/module_plugin_loader.gradle

```diff
@@ -9,13 +9,16 @@ import groovy.json.JsonSlurper
 
 def moduleProjectRoot = project(':flutter').projectDir.parentFile.parentFile
 
+def androidPlugins = null;
+String flutterModulePath = project(':flutter').projectDir.parentFile.getAbsolutePath()
 // Note: if this logic is changed, also change the logic in app_plugin_loader.gradle.
 def pluginsFile = new File(moduleProjectRoot, '.flutter-plugins-dependencies')
 if (pluginsFile.exists()) {
     def object = new JsonSlurper().parseText(pluginsFile.text)
     assert object instanceof Map
     assert object.plugins instanceof Map
     assert object.plugins.android instanceof List
+    androidPlugins = object.plugins.android
```

Comment:
I would move `object` definition to line 12 instead.

Reply:
The `object `was generated by `pluginsFile` which defined at line 15. Isn't the `object` must defined below `pluginsFile`?

Reply:
you can add `def object = null` to line 12. Then remove `def` from line 17. then use object in line 43, and line 45

Reply:
ok.